src/Umbraco.Web.UI.Client/src/views/common/dialogs/help.controller.js

```diff
@@ -35,7 +35,11 @@ angular.module("umbraco")
     	    });
 
     	    helpService.findVideos(rq).then(function(videos){
-    	    	$scope.videos = videos;
+    	        $scope.videos = videos;
+
+    	        angular.forEach($scope.videos, function (obj) {
+    	            obj.thumbnail = obj.thumbnail.replace(/(\.[\w\d_-]+)$/i, '_thumb$1');
+    	        });
```

Comment:
This just add insert "_thumb" before extension, e.g. filename_thumb.jpg
However it seems that not all video images have a "_thumb" image generated .. and if we only need the small thumbnails there is no need to request the large images?

![2015-02-04_22-37-39](https://cloud.githubusercontent.com/assets/2919859/6050291/a40a9632-acbe-11e4-94cc-fc8d969bb203.jpg)